smartmeter_datacollector/smartmeter/hdlc_dlms_parser.py

```diff
@@ -35,6 +37,9 @@ def __init__(self, cosem_config: CosemConfig, block_cipher_key: str = None) -> N
         self._cosem = cosem_config
 
     def append_to_hdlc_buffer(self, data: bytes) -> None:
+        if self._hdlc_buffer.getSize() > self.HDLC_BUFFER_MAX_SIZE:
+            LOGGER.info("HDLC byte-buffer > %i. Cleared.", self.HDLC_BUFFER_MAX_SIZE)
```

Comment:
Shouldn't this be a warning? And maybe mention in the log message that data has probably been lost.